sdk/Cargo.toml

```diff
@@ -9,9 +9,12 @@ license = "Apache-2.0"
 edition = "2018"
 
 [features]
-# On-chain program specific dependencies
+# "program" feature is a legacy feature retained to support v1.3 and older
+# programs.  New development should not use this feature.  Instead use the
+# solana-program-sdk crate
 program = []
```

Comment:
Don't think we can make a clear break here and remove this entirely?

Reply:
Nope, cause it'll break a downstream project that upgrades solana-sdk instead of migrating to solana-program-sdk